helm/organization-operator/values.yaml

```
@@ -29,3 +29,10 @@ serviceMonitor:
 global:
   podSecurityStandards:
     enforced: false
+
+# Metrics port
+ports:
+  metrics: 8080
+
+ciliumNetworkPolicy:
+  enabled: false
```

```suggestion
  enabled: false
```

Any reason why this is disabled by default?